v3/glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+* _channel_ : **TODO: find good description for channel**
+* _cloaking_ : method used to hides telehash traffic on the wire by randomizing all data sent
+* _Cipher Set (CS)_ : a collection of crypto algorithms with a given _cipher set id (CSID)_
+* _Cipher Set ID (CSID)_ : predefined hex number identifying a _cipher set (CS)_
+* _endpoint_ : a participant in the telehash network identified by a **(TODO: or multiple?)** _hashname_
+* _EVEN_ : the _endpoint_ with the numerically lower public key
+* _e3x_ : End-to-End Encrypted eXchange is a flexible encrypted exchange protocol
+* _exchange_ : **TODO: find good description for exchange**
```

Comment:
It's an encrypted session state between two endpoints, used to generate/process handshakes and create/manage channels?